omap-tutorial/README.md

```diff
@@ -139,6 +139,9 @@ Go to omap-tutorial folder.
 export LD_LIBRARY_PATH=/usr/local/lib
 
 test-platform ./PandaBoardES.xml
+ # Disable criteria check at XML load time in order to enable test without criteria.
+remote-process localhost 5001 setFailureOnFailedSettingsLoad false
```

Comment:
Should this not be disabled by default ? The criteria file is provided in this file, there is no reason the settings load should fail, isn't it ?